### PR TITLE
more int to isize cleanup in tests

### PR DESCRIPTION
**Pull request by compwron**
Sorry about the tininess of this change :/

### CODE REVIEW

**Comment by steveklabnik** [comment]
>  Sorry about the tininess of this change :/

Don't worry about it! This is a perfectly good patch. :smile: 

I think that these should be `i32`s and not `isize`s though. Mind changing it to that? r=me after.

**Comment by compwron** [comment]
r=steveklabnik

File: src/test/run-pass/fact.rs [hunk 1]
```diff
@@ -11,7 +11,7 @@
 
 
 
-fn f(x: int) -> int {
+fn f(x: i32s) -> i32s {
```
Comment (alexcrichton) [inline src/test/run-pass/fact.rs]: Did you mean `i32` instead of `i32s` here?

Comment (alexcrichton) [inline src/test/run-pass/fact.rs]: (and another spot down below)

**Comment by alexcrichton** [comment]
Closing due to inactivity, but feel free to reopen with a rebase!

**Pull request closed without merging**